src/main/scala/edu/gemini/gsp/sbtplugin/LucumaPlugin.scala

```diff
@@ -37,7 +36,6 @@ object LucumaPlugin extends AutoPlugin {
     lazy val lucumaPublishSettings = Seq(
       organization     := "edu.gemini",
       organizationName := "Association of Universities for Research in Astronomy, Inc. (AURA)",
-      startYear        := Some(2019),
```

Comment:
Why is this gone?

Reply:
It's not being used AFAIK since you customize the header text.